src/codegen/enums.rs

```diff
@@ -33,6 +34,7 @@ pub fn generate(env: &Env, root_path: &Path, mod_rs: &mut Vec<String>) {
             if enum_.glib_get_type.is_some() {
                 has_get_type = true;
             }
+            generate_display_trait = config.generate_display_trait || generate_display_trait;
```

Comment:
Maybe better `generate_display_trait |= config.generate_display_trait;`

Reply:
I'm shared on this: it doesn't bring much and makes code a bit more difficult to read. I don't have strong opinion about it though... @sdroege any opinion on this?

Reply:
I agree with @EPashkin